src/drug_discovery/external_tools/utils.py

```diff
@@ -23,30 +23,11 @@
 
 import os
 import pathlib
-from pathlib import Path
 
 from beartype import beartype
 import requests
 
 
-@beartype
-def count_atoms_in_pdb_file(pdb_file_path: str | Path) -> int:
-    """Count the number of atoms in a PDB file
-
-    Args:
-        pdb_file_path (str | Path): The path to the PDB file.
-
-    Returns:
-        int: The number of atoms in the PDB file.
-    """
-    from Bio.PDB import PDBParser
-
-    parser = PDBParser(QUIET=True)
-    structure = parser.get_structure("complex", str(pdb_file_path))
-
-    return sum(1 for _ in structure.get_atoms())
-
-
```

Comment:
this should be a method of protein, not a standalone function

Reply:
In my PDB parser, counts and lookups are part of an intermediate representation, tailored to the use case which, in that case, is 3d representation.